src/main/java/seedu/address/ui/AppointmentCard.java

```diff
@@ -68,11 +69,17 @@ public AppointmentCard(AppointmentView apptView, int displayedIndex) {
         note.setText("Notes: " + appt.getNote().toString());
         Timeline timeline = getTimeline();
         timeline.play();
+        //make appointment green depending on mark status
+        if (appt.getMark().isMarked) {
+            cardPane.setStyle("-fx-background-color: #33B864;");
```

Comment:
Can consider changing the colour to a constant if we intend to use it in other components. If not, this is fine!

Reply:
Good idea! Will make the changes